### PR TITLE
Updated domain invitation text content

### LINKED ISSUE #1047: Update text document for data migration domain invitations
### Issue Description

Email content for domain invitations sent during data migration will be updated with new verbiage as defined in #1034. [Content update here](https://docs.google.com/document/d/1bSy6lNkoImlO3rG7kzgZQy1nVubXmEDmmMHyhuOuH6g/edit#heading=h.gjdgxs). Note that there are comments for changes in wording based on whether the user has one or multiple domains.

### Acceptance Criteria
- [ ] Update the text document that the domain invitation migration script will use for its text body

### Additional Context (optional)

_No response_

### Issue Links

relates to: #1034

### PR DESCRIPTION
**Pull request by abroddrick**
## Ticket

Resolves #1047 

## Changes

-updated transition  domain invitation  email subject
- updated transition domain invitation content

## Setup

For designers:
Ask and I will send an email directly to you so you can see it just as a registrant will, please ping me in slack!

for others:
PREFACE: make sure transition domains doesn't have any emails you don't want being used in a test (aka docker-compose down then up to reset that database)
1.) In your local envirornment (src folder), run the following command to generate test transition_domains with emails you want
docker-compose exec app python manage.py generate_test_transition_domains -e your_email1,your_email2
OR just manually make some transition domains via django admin

2.) Then run the script to send emails:
docker-compose exec app python manage.py send_domain_invitations -s

#### Satisfied acceptance criteria and met development standards

- [ ] Met the acceptance criteria, or will meet them in a subsequent PR
- [ ] Created/modified automated tests
- [ ] Added at least 2 developers as PR reviewers (only 1 will need to approve)
- [ ] Messaged on Slack or in standup to notify the team that a PR is ready for review
- [ ] Changes to “how we do things” are documented in READMEs and or onboarding guide
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the assoicated migrations file has been commited.

#### Ensured code standards are met (Original Developer)

- [ ] All new functions and methods are commented using plain language
- [ ] Did dependency updates in Pipfile also get changed in requirements.txt?
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values

#### Validated user-facing changes (if applicable)

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)
- [ ] Add at least 1 designer as PR reviewer

#### Reviewed, tested, and left feedback about the changes

- [ ] Pulled this branch locally and tested it
- [ ] Reviewed this code and left comments
- [ ] Checked that all code is adequately covered by tests
- [ ] Made it clear which comments need to be addressed before this work is merged
- [ ] If any model was updated to modify/add/delete columns, makemigrations was ran and the assoicated migrations file has been commited.

#### Ensured code standards are met (Code reviewer)

- [ ] All new functions and methods are commented using plain language
- [ ] Interactions with external systems are wrapped in try/except
- [ ] Error handling exists for unusual or missing values
- [ ] (Rarely needed) Did dependency updates in Pipfile also get changed in requirements.txt?

#### Validated user-facing changes as a developer

- [ ] New pages have been added to .pa11yci file so that they will be tested with our automated accessibility testing
- [ ] Checked keyboard navigability
- [ ] Meets all designs and user flows provided by design/product
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers, the suggestion is to use ones that the developer didn't (check off which ones were used)
  - [ ] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

**Note:** Multiple code reviewers can share the checklists above, a second reviewers should not make a duplicate checklist

#### Verified that the changes match the design intention

- [x] Checked that the design translated visually
- [x] Checked behavior
- [ ] Checked different states (empty, one, some, error)
- [ ] Checked for landmarks, page heading structure, and links
- [ ] Tried to break the intended flow

#### Validated user-facing changes as a designer

- [ ] Checked keyboard navigability
- [ ] Tested general usability, landmarks, page header structure, and links with a screen reader (such as Voiceover or ANDI)

- [ ] Tested with multiple browsers (check off which ones were used)
  - [x] Chrome
  - [ ] Microsoft Edge
  - [ ] FireFox
  - [ ] Safari

- [ ] (Rarely needed) Tested as both an analyst and applicant user

### CODE REVIEW

**Comment by gabydisarli** [comment]
Are we able to de-link login.gov when it presents on the email side? On Gmail, it links login.gov in the header. 
Screenshot 2023-11-07 at 9 46 28 AM

**Review by michelle-rago — CHANGES_REQUESTED** [review]
This looks great, but I have some tweaks!! 

- Add a line break after "old system." before the numbered list for checking domain contacts.
- Change "However, in our new registrar, these contacts are all considered “domain managers.”" to "In our new registrar, these contacts are all considered “domain managers.”"
- Change "If you need help finding the email you used in the past" to "If you need help finding the email address you used in the past"

File: src/registrar/templates/emails/transition_domain_invitation.txt [hunk 1]
```diff
@@ -1,20 +1,42 @@
...
+You can’t use your old credentials to access the new registrar. Access is now managed through Login.gov, a simple and secure process for signing into many government services with one account. Follow these steps to create your Login.gov account <https://login.gov/help/get-started/create-your-account/>. 
+
+When creating an account, you’ll need to provide the same email address you used to log in to the old registrar. That will ensure your domains are linked to your Login.gov account. 
+
+If you need help finding the email address you used in the past, let us know in a reply to this email.
+
+
+CHECK YOUR .GOV DOMAIN CONTACTS
+
+This is a good time to check who has access to your .gov domain{% if domains|length > 1 %}s{% endif %}. The admin, technical, and billing contacts listed for your domain{% if domains|length > 1 %}s{% endif %} in our old system also received this email. In our new registrar, these contacts are all considered “domain managers"
```
Comment (gabydisarli) [inline src/registrar/templates/emails/transition_domain_invitation.txt:23]: needs a period at end: "domain managers**.**"

**Comment by gabydisarli** [comment]
> Are we able to de-link login.gov when it presents on the email side? On Gmail, it links login.gov in the header. Screenshot 2023-11-07 at 9 46 28 AM

was this something we can do?

**Comment by Katherine-Osos** [comment]
@gabydisarli Re: unlinking Login.gov. There's a ticket to do some research around that: #1072

File: src/registrar/templates/emails/transition_domain_invitation.txt (hunk 1 above)
Comment (Katherine-Osos) [reply src/registrar/templates/emails/transition_domain_invitation.txt:23]: Nice catch!

**Comment by abroddrick** [comment]
@gabydisarli  no we can't de link at the moment, there is another ticket to investigate doing this, but requires going through gmail's docs on how to do this

**Review by michelle-rago — CHANGES_REQUESTED** [review]
Sorry, Alysia! Can you add a period at the end of this sentence? There should be a period after the >.

1. Once you have your Login.gov account, sign in to the new registrar at <https://manage.get.gov/>

**Comment by gabydisarli** [comment]
> Agreed with Katherines comment, that is a whole separate ticket. Adding that period in

ah okay, yes i remember some discussion happening about it on Slack but was unaware it wasn't solved.

**Comment by gabydisarli** [comment]
Noticed weird styling difference of the text here on dark mode on mobile.

IMG_8992

File: src/registrar/templates/emails/transition_domain_invitation.txt (hunk 1 above)
Comment (gabydisarli) [reply src/registrar/templates/emails/transition_domain_invitation.txt:23]: @abroddrick sorry, also noticing the use of a prime/hatch mark at the end of domain managers when it should be a quotation mark. ”

**Comment by gabydisarli** [comment]
not sure if this will work but it looks like some formatting is getting carried over on copy + paste. Usually I do Shift + V to paste without formatting.

File: src/registrar/templates/emails/transition_domain_invitation.txt (hunk 1 above)
Comment (abroddrick) [reply src/registrar/templates/emails/transition_domain_invitation.txt:23]: once again this bites me. any chance you know a hot key to make sure it's a quotation mark?

**Comment by abroddrick** [comment]
> not sure if this will work but it looks like some formatting is getting carried over on copy + paste. Usually I do Shift + V to paste without formatting.

I can do this on all of the text, but I also can't replicate what you're seeing at all. My system is in darkmode and gmail defaults to light still but when I change gmail settings to dark mode I get this: 
image
did you change anything else on your settings?

File: src/registrar/templates/emails/transition_domain_invitation.txt (hunk 1 above)
Comment (gabydisarli) [reply src/registrar/templates/emails/transition_domain_invitation.txt]: option/alt + shift + '

**Comment by gabydisarli** [comment]
> not sure if this will work but it looks like some formatting is getting carried over on copy + paste. Usually I do Shift + V to paste without formatting.

False alarm. This is a [feature of Gmail ](https://support.taguchi.com.au/knowledge-base/campaigns-and-activities/content/why-is-the-text-purple-in-gmail.html#:~:text=It%20is%20caused%20by%20Gmail,purple%20coloured%20text%20will%20disappear.)where it's highlighting the duplicate text from a threaded email with a different color. In this case, the paragraph that's colored differently is doing so because it's not duplicate text (because you added the period.) TIL!

**Comment by gabydisarli** [comment]
> This looks great, but I have some tweaks!!
> 
> * Add a line break after "old system." before the numbered list for checking domain contacts.
> * Change "However, in our new registrar, these contacts are all considered “domain managers.”" to "In our new registrar, these contacts are all considered “domain managers.”"
> * Change "If you need help finding the email you used in the past" to "If you need help finding the email address you used in the past"

@abroddrick I think the most recent email I received doesn't have the 2nd change Michelle requested here, I still see the word "however."

**Comment by abroddrick** [comment]
> > not sure if this will work but it looks like some formatting is getting carried over on copy + paste. Usually I do Shift + V to paste without formatting.
> 
> False alarm. This is a [feature of Gmail ](https://support.taguchi.com.au/knowledge-base/campaigns-and-activities/content/why-is-the-text-purple-in-gmail.html#:~:text=It%20is%20caused%20by%20Gmail,purple%20coloured%20text%20will%20disappear.)where it's highlighting the duplicate text from a threaded email with a different color. In this case, the paragraph that's colored differently is doing so because it's not duplicate text (because you added the period.) TIL!

OMG Thank goodness, I was like "What happened?".

**Comment by abroddrick** [comment]
> @abroddrick I think the most recent email I received doesn't have the 2nd change Michelle requested here, I still see the word "however."

Thanks for catching that, it should be fixed on the newest email you just recieved! @gabydisarli

**Review by michelle-rago — APPROVED** [review]
The email looks good to me! If we don't end up pulling in the AO info or the organization address, we will need to update the text.

**Pull request merged**